lib/backends/vault-backend.js

```diff
@@ -0,0 +1,97 @@
+'use strict'
```

Comment:
you use `var` in many places where `const` or `let` will work.  Can you convert to `const` and `let`?

Reply:
Sure thing - switching to consts as much as possible!